README.md

```diff
@@ -105,41 +145,43 @@ buildscript {
 }
 ```
 
-Don't forget to apply the plugin:
+Then you can `apply plugin` (example in Groovy):
 
 ```gradle
 apply plugin: 'kotlin' // or 'kotlin-multiplatform' for multiplatform projects
-apply plugin: 'kotlinx-serialization'
+apply plugin: 'org.jetbrains.kotlin.plugin.serialization'
```

Comment:
Again, not tested assumption: `apply plugin: 'kotlinx-serialization'` should still work. Probably, it doesn't work for Kotlin DSL but works for Gradle DSL. Or works when dependency declared as  `classpath "org.jetbrains.kotlin:kotlin-serialization:$kotlin_version"` but not as `classpath(kotlin("serialization", version = kotlinVersion))`... Gradle is somewhat complicated mechanism.

Reply:
There is no difference between the two `classpath` syntaxes you mentioned - the `kotlin()` extension function is here again a convenience function which just appends the proper group, module prefix and version number:
```
fun DependencyHandler.kotlin(module: String, version: String? = null): Any =
    "org.jetbrains.kotlin:kotlin-$module${version?.let { ":$version" } ?: ""}"
```

Reply:
Regarding the first part of your question, the old plugin id
`apply plugin: 'kotlinx-serialization'`
indeed still does work. I will restore the example.